conda/recipes/libcudf/meta.yaml

```
@@ -70,17 +84,26 @@ outputs:
       run_exports:
         - {{ pin_subpackage("libcudf", max_pin="x.x") }}
       ignore_run_exports_from:
+        {% if cuda_major == "11" %}
+        - {{ compiler('cuda11') }}
+        {% else %}
         - {{ compiler('cuda') }}
+        {% endif %}
```

Do we want to limit this to CUDA 11 (like [was done in RMM]( https://github.com/rapidsai/rmm/blob/branch-23.08/conda/recipes/rmm/meta.yaml#L34-L37 ))?

```suggestion
      ignore_run_exports_from:
        {% if cuda_major == "11" %}
        - {{ compiler('cuda11') }}
        {% endif %}
```